src/todoItem.js

```diff
@@ -0,0 +1,51 @@
+import { getStorage, setStorage } from './localStorageData';
+
+const localData = getStorage();
+
+let idCount = localData[0];
+
+// list default value creation
+const listStored = localData[1];
+let list = {};
+if (localData[1] !== null) {
```

Comment:
A variable has been created on line  `8` for this, please use this instead [`OPTIONAL`].